src/components/pages/home.jsx

```diff
@@ -1,45 +1,59 @@
 import {Link} from "react-router"
-import React from "react"
+import React, {PropTypes} from "react"
 
 import BreadCrumb from "../breadcrumb"
+import ExternalLink from "../external-link"
```

Comment:
Not a React expert, but I don't see this used further in the file. Is it on purpose ?